tests/tools/test_structsearch.py

```diff
@@ -62,3 +66,36 @@ def test_create_remaining_models(load_example_model_for_test):
     model = load_example_model_for_test("pheno")
     models = create_remaining_models(model, ests)
     assert len(models) == 12
+
+
+def test_pkpd(load_model_for_test, testdata):
+    ests = [0, 1, 2, 3, 4]
```

Comment:
Maybe it would make sense here to use parameter estimates from a `ModelfitResults` object since that's how it's used in the code?